quantization/plot_quantization_s2.py

```diff
@@ -0,0 +1,54 @@
+"""
+Plot the result of optimal quantization of the von Mises Fisher distribution
+on the sphere
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import geomstats.visualization as visualization
+import quantization.optimal_quantization as oq
+
+from geomstats.hypersphere import Hypersphere
+
+SPHERE2 = Hypersphere(dimension=2)
+
+
+def main():
+    # generate points from the von Mises Fisher distribution
+    n_points = 1000
+    points = SPHERE2.random_von_mises_fisher(kappa=10, n_samples=n_points)
+
+    # find the 5 centers that best approximate the points
+    n_centers = 5
+    centers, weights, clustering, n_steps = oq.optimal_quantization(
+                points, n_centers, space='S2'
+                )
+    plt.figure(0)
+    ax = plt.subplot(111, projection="3d", aspect="equal")
+    visualization.plot(centers, ax, space='S2', c='r')
+    plt.show()
+
+    # the Voronoi diagram associated to the centers yields a clustering
+    clusters = {}
+    for i in range(n_centers):
+        clusters[i] = points[clustering == i]
+
+    sphere = visualization.Sphere()
+
+    plt.figure(1)
+    ax = plt.subplot(111, projection="3d", aspect="equal")
+    color = np.random.rand(n_centers, 3)
+    ax.plot_wireframe(sphere.sphere_x,
```

Comment:
Use methods of the sphere: `sphere.draw`, `sphere.add_points`, `sphere.draw_points`, etc
https://github.com/geomstats/geomstats/blob/master/geomstats/visualization.py#L94

Reply:
I have tried to use them in the best way possible, however it would be more satisfactory if the sphere was not drawn as many times as the number of clusters (I need each cluster to be drawn in a different color and so I repeat ```sphere.draw``` ```n_clusters``` times). Maybe this could be fixed by creating a new list of points each time ```add_points``` is called, that could be plotted in a different color (or some other way) ?

Also, I would recommend to change the ```alpha=0.5``` in ```draw``` to a lower value such as ```alpha=0.2``` so that the points are more visible, or to make it into a adjustable parameter.

Reply:
Never mind, the sphere is not drawn when using ```sphere.draw_points``` instead of ```sphere.draw```. Sorry about that !